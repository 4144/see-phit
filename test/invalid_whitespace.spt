R"*(
< HTML>
  <DIV> 
    Halloo
  </DIV>
  
  <DIV> 
    Guten tag
  </DIV >
  <IMG/>
  <IMG/>
  <P> 
    Hi
  </P>
</HTML>  
)*"_html; 