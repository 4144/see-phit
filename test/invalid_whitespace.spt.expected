       ParseError("Expecting a tag name after <");
